internal/pkg/crypto/ca.go

```diff
@@ -0,0 +1,133 @@
+package crypto
```

Comment:
I think that the  package  name is a bit confusing. Usually crypto is used for packages that deal with cryptography algorithms, in this case I think this package can be merged with the certificates package

Reply:
That's right. I'll push the changes fixing that soon